### PR TITLE
osp page

### PR DESCRIPTION
**Pull request by yingguangchu**

### CODE REVIEW

File: docs/osp.md [hunk 1]
```diff
@@ -9,13 +9,14 @@ parent: Co-simulation Platforms
...
+With common tools, standards and specifications, OSP seeks to foster co-simulation collaboration in industry.
 
-There is currently support for FMI 2.0 (including FMI 2.0.1) and FMI 1.0. FMI 3.0 will be supported when this standard becomes operational. The structure of the co-simulated system can be defined using the SSp standard or the new OSP System Structure format: OSP-IS.
-
-The main benefit introduced with OSP-IS is the possibility to define FMU interconnections at a higher level, i.e. variable group connections rather than scalar connections. Additionally, it enables automatic unit conversions and provides the a priori validation of the system configuration utilizing an ontology that models the connection types.
+The OSP currently supports for FMI 2.0 and FMI 1.0. FMI 3.0 will be supported when this standard becomes operational. The structure of the co-simulated system can be defined using the SSp standard or the new OSP System Structure format: OSP-IS.
```
Comment (markaren) [inline docs/osp.md]: SSp -> SSP

File: docs/osp.md [hunk 2]
```diff
@@ -9,13 +9,14 @@ parent: Co-simulation Platforms
...
+The main benefit introduced with OSP-IS is the possibility to define FMU interconnections at a higher level, i.e. variable group connections rather than scalar connections. Additionally, it enables automatic unit conversions and provides a priori validation of the system configuration utilizing an ontology that models the connection types.
```
Comment (markaren) [inline docs/osp.md]: > Additionally, it enables automatic unit conversions

So does SSP, but OSP does not actually support it for any of the formats yet I believe.

Comment (yingguangchu) [reply docs/osp.md]: it has two functions currently. Have used in the ssd file.

Comment (markaren) [reply docs/osp.md]: mhh? I'm saying libcosim currently does not support automatic unit conversion. Neither for SSP or OSP-IS. That is specifying units within the FMI modelDescription.xml or the SystemStructureDefinition.ssd. Correct me if I'm wrong about supporting this for OSP-IS. It's planned for sure, but not currently in.

Comment (yingguangchu) [reply docs/osp.md]: No you are right. Not automatic conversions right now.  I'll fix this.

Comment (markaren) [reply docs/osp.md]: No, don't change it. You are describing the standard, not the implementation. My bad

Comment (yingguangchu) [reply docs/osp.md]: Well that would be misleading before it be implemented. I think it's better to remove "automatic" for now.

**Review by markaren — APPROVED** [review]

**Pull request merged**